### PR TITLE
Fixed: E-mail verification is sometimes called confirmation

### PR DESCRIPTION
**Pull request by rahulm24**
Jira Ticket: MBS-8699
Changed all instances of the usage of confirm in the context of emails to verify/verified/verification

### CODE REVIEW

**Comment by mwiencek** [comment]
@rahulm24 Did you intend to close the pull request? (You don't have to open a new one to make changes, simply push any new commits here. :))

**Comment by mwiencek** [comment]
Thanks! I think this fixes all the user visible ones. There's a couple more I think we can change, though: the error message "a confirmed email address is required" in lib/MusicBrainz/Server/Controller/WS/js/Edit.pm, plus the associated test in t/lib/t/MusicBrainz/Server/Controller/WS/js/Edit.pm (including the test description).

There are other places throughout the code we *could* change, though I'm less certain if these are worth the effort (@reosarevok @yvanzo ?):

1. Renaming the file account/sso/DiscourseUnconfirmedEmailAddress.js
2. Renaming the file t/lib/t/MusicBrainz/Server/Controller/UnconfirmedEmailAddresses.pm
3. Renaming the subroutine `_check_for_confirmed_email` in lib/MusicBrainz/Server/Controller/User.pm
4. Renaming `email_confirmation_date` in lib/MusicBrainz/Server/Entity/Editor.pm and throughout the code (check `git grep email_confirmation_date`), plus comments in that file. Though this currently matches up with the SQL column name (`email_confirm_date`).
5. Similarly renaming `has_confirmed_email_address` (check `git grep` again).

**Comment by rahulm24** [comment]
please ignore previous this commit  ^

File: root/user/UserProfile.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ const UserProfileInformation = withCatalystContext(({
     $c.user && !$c.user.has_confirmed_email_address) ? (
       <strong>
         {l(`Your homepage and biography will not show
-            until you have confirmed your email.`)}
+            until you have verified your email.`)}
```
Comment (yvanzo) [inline root/user/UserProfile.js]: Other occurrences are fine, and I may be wrong, but I think in this specific case the wording was correct:

* MusicBrainz _verify_ your email address (by sending an email to you).
* You _confirm_ your email address to MusicBrainz (by clicking the link in the email).

That is, you don’t _verify_ your own email address, you confirm it to MusicBrainz.

Comment (reosarevok) [reply root/user/UserProfile.js]: I suspect it's confusing to have both words. If you want to be more precise, what about "until you have completed the email verification process"?

Comment (reosarevok) [reply root/user/UserProfile.js]: Since I can't edit this PR, merged and will add a commit changing that string.

**Pull request merged**